src/u128_support.rs

```diff
@@ -0,0 +1,46 @@
+use byteorder;
+use prelude::*;
+
+impl Uuid {
+    /// Creates a new [`Uuid`] from a `u128` value.
+    ///
+    /// To create a [`Uuid`] from `u128`s, you need `u128` feature enabled for this crate.
+    ///
+    /// [`Uuid`]: ../struct.Uuid.html
+    #[inline(always)]
```

Comment:
Usually `inline(always)` isn't ideal. A simple `inline` (without the `always`) would be better. Did you have a case where this method wasn't being inlined?